File: gcd.cpp
#include <iostream>
using namespace std;

int gcd(int num1, int num2) {
    if (num2 == 0) {
        return num1;
    }
    return gcd(num2, num1 % num2);
}

int main() {
    int num1, num2;
    cout << "Enter first number: ";
    cin >> num1;
    cout << "Enter second number: ";
    cin >> num2;
    int result = gcd(num1, num2);
    cout << "The GCD of " << num1 << " and " << num2 << " is: " << result << endl;
    return 0;
}

File: gcd.js
function gcd(num1, num2) {
    if (num2 === 0) {
        return num1;
    }
    return gcd(num2, num1 % num2);
}

let num1 = parseInt(prompt("Enter first number: "));
let num2 = parseInt(prompt("Enter second number: "));
let result = gcd(num1, num2);
console.log(`The GCD of ${num1} and ${num2} is: ${result}`);
